test/test_prims.py

```diff
@@ -176,6 +176,25 @@ def func(a, b, c):
         out = execute(gm, a, a, a, executor="nvfuser")
         self.assertEqual(out, (a, a, a))
 
+    @onlyCUDA
+    @skipCUDAIfRocm
+    def test_nvfuser_rand_like_fusion(self, device):
+        from torch._prims.context import TorchRefsNvfuserCapabilityMode
+        from torch.fx.experimental.proxy_tensor import make_fx
+        from torch._prims.executor import execute
+
+        a = torch.randn(3, 3, device=device)
+
+        def func(a):
+            return torch.rand_like(a)
+
+        with TorchRefsNvfuserCapabilityMode():
+            gm = make_fx(func)(a)
+
+        # Should pass with partitioned executor
```

Comment:
nit: strictly_nvfuser is not a partitioned executor 😄 Everything that's in `gm` is sent to nvFuser's execution in this case.

Reply:
Removed the comment.